intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -1366,3 +1366,25 @@ \subsection{Generate your own e-mail notifications}
 \end{prompt}
 
 will send an e-mail to john.smith@example.com when the job begins.
+
+\section{Running a job after another job}
+
+If you submit two jobs expecting that should be run one after another (for example because
+the first generates a file the second needs), there might be a problem as they might both be run at the same time.
+
+So the following example might go wrong:
+
+\begin{prompt}
+%\shellcmd{qsub job1.sh}%
+%\shellcmd{qsub job2.sh}%
+\end{prompt}
+
+You can make jobs that depend on other jobs. This can be useful for breaking up large jobs
+in smaller jobs that can be ran in a pipeline. The following example will submit 2 jobs,
+but the second job (\lstinline|job2.sh|) will be held (\lstinline|H| status in \lstinline|qstat|)
+until the first job successfully completes. If the first job fails, the second will be cancelled.
+
+\begin{prompt}
+%\shellcmd{FIRST\_ID=\$(qsub job1.sh)}%
+%\shellcmd{qsub -W depend=afterok:\$FIRST\_ID job2.sh}%
+\end{prompt}
```

Comment:
Elaborate a little bit: explain what `afterok` means, and also mention the alternatives `afterany` and `afternotok`, see https://docs.loni.org/wiki/PBS_Job_Chains_and_Dependencies (don't refer to that link, but it can serve as an inspiration).